utils/util.js

```diff
@@ -0,0 +1,24 @@
+export const formatPhone = phone => {
+  if (!phone) return;
+  if (phone[0] === '(') return phone;
```

Comment:
might want to use a regex.test here to make sure it follows the entire format. i believe that would look something like:

```
const re = /\([0-9]{3}\) [0-9]{3}-[0-9]{4}/;
re.test(phone);
```

Reply:
I should have definitely used regex here instead of doing the minimum based off the contact data given.